### PR TITLE
fix: Updated copy change on Name Change modal

### PR DESCRIPTION
**Pull request by rijuma**
### Description

Just a copy change on the Name Change modal. This is part of the New IDV vendor work https://github.com/openedx/platform-roadmap/issues/367.

Old copy:
> Enter your name as it appears on your unexpired student, work, or government-issued identification card.

New copy:
> Enter your name as it appears on your identification card.

#### How Has This Been Tested?

Tested locally.

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/edx-infinity** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: src/account-settings/AccountSettingsPage.messages.jsx [hunk 1]
```diff
@@ -143,7 +143,7 @@ const messages = defineMessages({
   },
   'account.settings.field.name.verified.verification.alert': {
     id: 'account.settings.field.name.verified.verification.help',
-    defaultMessage: 'Enter your name as it appears on your unexpired student, work, or government-issued identification card.',
+    defaultMessage: 'Enter your name as it appears on your identification card.',
```
Comment (Zacharis278) [inline src/account-settings/AccountSettingsPage.messages.jsx:146]: looks to me like this is an unused duplicate. Maybe it should be removed?

**Review by Zacharis278 — APPROVED** [review]

**Review by MichaelRoytman — APPROVED** [review]
This looks good to me - agree with Zach about the duplicate.

Could you link to the https://github.com/openedx/platform-roadmap/issues/367 issue in your pull request description, briefly explain that we are doing this to generalize the copy, and request the Infinity team for a review, please?

File: src/account-settings/AccountSettingsPage.messages.jsx (hunk 1 above)
Comment (rijuma) [reply src/account-settings/AccountSettingsPage.messages.jsx]: Removed.

**Review by ayesha-waris — APPROVED** [review]
PR LGTM

**Pull request merged**